src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -49,15 +44,43 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
         return hass.states[c.entity].state !== c.state_not;
       });
 
-    if (visible && this.card) {
-      this.card.hass = hass;
+    if (this._card) {
+      this._card.hass = this._hass;
     }
 
     this.style.setProperty("display", visible ? "" : "none");
   }
 
-  public getCardSize() {
-    return computeCardSize(this.card);
+  public getCardSize(): number {
+    return computeCardSize(this._card);
+  }
+
+  public setConfig(config: Config): void {
+    if (
+      !config.card ||
+      !config.conditions ||
+      !Array.isArray(config.conditions) ||
+      !config.conditions.every(
+        (c) =>
+          c.hasOwnProperty("entity") &&
```

Comment:
What is the reason behind using `hasOwnProperty`?

Reply:
TypeScript linter doesn't like the check c.entity `'string' is not assignable to type 'boolean'`. Debated using `!!c.entity` but `hasOwnProperty` felt more proper. 🤷‍♂️

Reply:
Idk the proper syntax here. Sorry. I was just wondering. You could as type cast but I think what you have or `!!` is better

Reply:
alternative is to use `('entity' in c)`.